app/views/idv/address/new.html.erb

```diff
@@ -28,9 +28,11 @@
     ) do |f| %>
   <%= render ValidatedFieldComponent.new(
         form: f,
+        id: 'address1',
         name: :address1,
         label: t('idv.form.address1'),
         hint: @presenter.address_line1_hint,
+        hint_class: @presenter.hint_class,
```

Comment:
I think it may not be necessary to add this as a new standalone argument, since ValidatedFieldComponent passes through all additional options to SimpleForm. I think you could do something like this instead:

```suggestion
        hint_html: { class: @presenter.hint_class },
```

Reply:
Did you also remove the ValidatedFieldComponent revisions? I could imagine with those as implemented, the `hint_html` class would be overridden. But reverting those changes should work. #8232